app/models/katello/glue/pulp/pulp_content_unit.rb

```diff
@@ -32,6 +32,7 @@ def with_identifiers(ids)
       end
 
       def in_repositories(repos)
+        repos = Array(repos)
```

Comment:
Array() will return the same array if its passed an array, and a blank array if passed nil